Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py

```diff
@@ -10,18 +11,22 @@ def read_qr_code(filename: str) -> str:
     img = cv2.imread(filename)
```

Comment:
Just something to mention, cv2 is utilizing c library calls so if those libraries issue a warning the task will be considered a failure. Warnings are written to stderr so xsoar thinks the task failed.

I faced this issue with libpng.

I would recommend redirecting stderr here

from wurlitzer import pipes

with pipes() as (out, err):
    cv2.imread(filename)
    
demisto.info(err.read())

Reply:
Thanks for the tip! @jlevypaloalto will look into it

Reply:
Hi @enes-oezdemir, thanks for your input!
It would be helpful if you could provide an example of a warning in cv2 that concerns our use-case.

Reply:
here is a warning when runnign cv2.imread() with an image in my environment

libpng warning: bKGD: invalid

Reply:
![Untitled](https://github.com/demisto/content/assets/151725756/d5f045cb-c9d5-4309-b57b-061d99cd1b2d)

For example, this is a file, I know that this is not a valid QR code but cv2.imread will complain even when loading the image (cv2.imread). This behavior might result in a task error even if you have a valid QR code

Reply:
Thanks for your help @enes-oezdemir! 😇
I'll add the context manager to the code.